LibGUI/GWindow.cpp

```diff
@@ -53,6 +53,7 @@ void GWindow::show()
 {
     if (m_window_id)
         return;
+    dbgprintf("Window Type: %d\n", m_window_type);
```

Comment:
It's fine to add dbgprintf() here and there, but it's nice if you make them include some context so the stream of text makes some sense.
In this case, I would say something like "GWindow::show(): Window type: %d\n"